NEWS.md

```diff
@@ -4,6 +4,19 @@ Trixi.jl follows the interpretation of [semantic versioning (semver)](https://ju
 used in the Julia ecosystem. Notable changes will be documented in this file
 for human readability.
 
+## Changes in the v0.6 lifecycle
+
+#### Added
+
+#### Changed
+
+- The default wave speed estimate using within `flux_hll` is now `min_max_speed_davis` 
+  instead of `min_max_speed_naive`.
```

Comment:
Since this is a breaking change, we need to move it to v0.7. Could you please update this part so it becomes clear by introducing a section like "## Changes when updating to v0.7 from v0.6.x"?